assets/js/react/GameStateEditor/reducers/entityForm.js

```diff
@@ -7,27 +8,41 @@ import {
   ADD_ENTITY_SUCCESS,
   ADD_ENTITY_FAILURE,
 } from "../actions/entityForm"
+import { EDIT_ENTITY, CLOSE_EDIT_PANE } from "../../actions/app"
+import { convertBehaviorsToHash } from "../selectors/entityForm"
 
 export const defaultState = {
   availableBehaviors: [],
   entity: {},
 }
 
 export default function(state = defaultState, action) {
+  const data = camelizeKeys(action.data)
   switch (action.type) {
+    case CLOSE_EDIT_PANE:
+      return mergeDeepRight(state, { entity: {} })
+
+    case EDIT_ENTITY:
+      if (data) {
```

Comment:
This event is sent from the `gameConsoleSaga` when someone types `/edit entityname`.  This is where we grab that data off the channel and stick it into the redux store which fills in the entityForm.  Additionally the app reducer is listening for `EDIT_ENTITY` so it can set the edit pane open.